browser/brave_content_browser_client.cc

```diff
@@ -911,7 +912,8 @@ bool BraveContentBrowserClient::WillCreateURLLoaderFactory(
   use_proxy |= ChromeContentBrowserClient::WillCreateURLLoaderFactory(
       browser_context, frame, render_process_id, type, request_initiator,
       std::move(navigation_id), ukm_source_id, factory_receiver, header_client,
-      bypass_redirect_checks, disable_secure_dns, factory_override);
+      bypass_redirect_checks, disable_secure_dns, factory_override,
+      navigation_response_task_runner);
```

Comment:
it seems we also need to reflect this change at `BraveProxyingURLLoaderFactory::InProgressRequest::ContinueToStartRequest`, similar to 

https://chromium.googlesource.com/chromium/src/+/7c0169035b5bcb96cd1e5a2b30fbb4577350c665%5E%21/#F24

Reply:
@iefremov, please take a look at https://github.com/brave/brave-core/pull/18808/commits/f063964e2fc5ecc8e479b74a57170e02c13a3fe9